docs/source/contributing/index.md

```diff
@@ -1,11 +1,12 @@
 ---
 html_meta:
-  'description': ''
-  'property=og:description': ''
-  'property=og:title': ''
-  'keywords': ''
+  "description": "Volto contributing guidelines"
+  "property=og:description": "Volto contributing guidelines"
+  "property=og:title": "Contributing Guidelines"
+  "keywords": "Contribute,Volto,Frontend,Github"
 ---
 
+(contributing-index-label)=
 # Design principles
```

Comment:
We need to discuss this structure as it is confusing. The OpenGraph data adds to the confusion. I would suggest clearing the OG values for now on these two pages until we can sort out where things should go.

- [Design principles](https://6.dev-docs.plone.org/volto/contributing/index.html)
- [Guidelines](https://6.dev-docs.plone.org/volto/contributing/guidelines.html)

My suggestions:

1. "Design principles" page should not be under a `contributing` URL. I think it should be moved to either the top level at `/design-principles.md` or under [Developer Guidelines](https://6.dev-docs.plone.org/volto/developer-guidelines/index.html) at `/developer-guidelines/design-principles.md`. I lean toward the latter.
2. Guidelines should be renamed to "Contributing Guidelines", and moved `/contributing.md` at the root level.

This needs to be discussed amongst the Volto team.